### DISCUSSION TITLE
How to import contracts in personal repo

### DISCUSSION THREAD

**Discussion by ankitchiplunkar** [OpenZeppelin/cairo-contracts#167] [Q&A] [answered]
Hi How can I import these contracts in personal repo?
In solidity I just installed the package and was able to use the contracts, is there something similar available in cairo-contracts?

**Comment by martriay** [comment, marked as answer]
Yes, [this is coming](https://github.com/OpenZeppelin/cairo-contracts/issues/125) with our upcoming [v0.1.0 release](https://github.com/OpenZeppelin/cairo-contracts/projects/2)!
